src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -88,7 +151,7 @@ internal bool CanCreateModel(MutableObjectBinderContext context)
             // recursion.
             //
             // First, we want to make sure this object is allowed to come from a value provider source as this binder
-            // will always include value provider data. For instance if the model is marked with [FromBody], then we
+            // will only include value provider data. For instance if the model is marked with [FromBody], then we
```

Comment:
While fixing this, have similar issues in the page-long comment at [lines 192-215](https://github.com/aspnet/Mvc/blob/momb/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs#L192-L215) e.g. `may include ALL value providers`.